contracts/tether/Ante_USDTPegTest.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.8.0;
+
+import "../interfaces/IERC20.sol";
+import "../AnteTest.sol";
+import "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
+
+// Ante Test to check USDT remains > 0.90
+contract AnteUSDTPegTest is AnteTest("USDT is above 90 cents on the dollar") {
+    // https://etherscan.io/token/0xdAC17F958D2ee523a2206206994597C13D831ec7
+    address public constant TetherAddr = 0xdAC17F958D2ee523a2206206994597C13D831ec7;
+
+    AggregatorV3Interface internal priceFeed;
+
+    /**
+     * Network: Mainnet
+     * Aggregator: USDT/USD
+     * Address: 0xEe9F2375b4bdF6387aa8265dD4FB8F16512A1d46
```

Comment:
hi. forgive me if this question has an obvious answer, i'm new to solidity.

is this the correct address?
this resolves to https://data.chain.link/ethereum/mainnet/stablecoins/usdt-eth
https://etherscan.io/address/0xee9f2375b4bdf6387aa8265dd4fb8f16512a1d46

isn't that usdt/eth?
wouldn't we instead want to use `0x3e7d1eab13ad0104d2750b8863b489d65364e32d`?
https://data.chain.link/ethereum/mainnet/stablecoins/usdt-usd

Reply:
@ok1a After checking the addresses on etherscan and the `lastestRoundData` function there, it would appear you're correct. If you'd like, you can create a PR against this so that it can be recorded as a PR?

Reply:
@j-nia Done! https://github.com/antefinance/ante-community-tests/pull/27